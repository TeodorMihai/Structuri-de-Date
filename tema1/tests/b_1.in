1 10
::w 5 0 AAAA
3
::m 1 30 0001 0002 0003 0004 0005 0006 .
4
::m 1 31 2
2
::m 2 0 AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF AAAA BBBB CCCC DDDD FFFF .
10
::m 2 60 2
2
::m 9 6100 2
1
::m 2 0 2
1
::m 5 5 0001 0002 0003 0004 0005 0006 0007 .
1
::m 5 6 1
1
::e