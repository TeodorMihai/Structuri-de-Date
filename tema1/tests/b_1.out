	0002
	0003
	AAAA
	BBBB
	0000
	0000
	AAAA
	BBBB
	0002
	2.53 1.82 1.45 1.47
